cmake/mlir_fetch.cmake.in

```diff
@@ -46,7 +46,7 @@ ExternalProject_Add(
     CMAKE_GENERATOR_TOOLSET ${CMAKE_GENERATOR_TOOLSET}
 
     BUILD_COMMAND ${CMAKE_COMMAND} ../llvm -DLLVM_BUILD_EXAMPLES=ON -DLLVM_TARGETS_TO_BUILD=host -DLLVM_ENABLE_RTTI=ON -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
-    COMMAND ${CMAKE_COMMAND} --build . --target check-mlir -- -j8 
+    COMMAND ${CMAKE_COMMAND} --build . --target check-mlir -- -j
```

Comment:
I'll try to find time to move this to the build instead of the configuration, as suggested, but it could take me some time. This is helping MLIR CI testing to the point that we could enable it for any PR so, please, consider if we can go with this in the mean time. No rush, though. We can wait if you think this is going to be problematic in the short term.